src/utils.ts

```diff
@@ -0,0 +1,81 @@
+import { ethers, Contract, BigNumber } from "ethers";
+import { Finding, FindingSeverity, FindingType, Alert } from "forta-agent";
+import { L2_FUNCTION_SIGNATURE, L2_TOKEN_ADDRESS_MAKER_DAO, PreviousBalances } from "./constants";
+
+export const getL1Finding = async (
+  daiContract: Contract,
+  blockNumber: number,
+  escrowAddressArbitrum: string,
+  escrowAddressOptimism: string,
+  previousBalances: PreviousBalances
+): Promise<Finding[]> => {
+  // Return type changed to Promise<Finding[]>
+  const arbitrumBalance: BigNumber = await daiContract.balanceOf(escrowAddressArbitrum, {
+    blockTag: blockNumber,
+  });
+  const optimismBalance: BigNumber = await daiContract.balanceOf(escrowAddressOptimism, {
+    blockTag: blockNumber,
+  });
```

Comment:
Although this works already, a small suggestion is to use `await Promise.all(..)` to group the calls and make it faster